### PR TITLE
🌐 Add Russian translation for `docs/ru/docs/tutorial/path-params.md`

### PR DESCRIPTION
**Pull request by AGolicyn**

### CODE REVIEW

File: docs/ru/docs/tutorial/path-params.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    Возможно, вам понадобится, чтобы параметр содержал `/home/johndoe/myfile.txt` с ведущим слэшем (`/`).
+
+    В этом случае URL будет таким: `/files//home/johndoe/myfile.txt`, с двойным слэшем (`//`) между `files` и `home`.
+
+## Резюме
+Используя **FastAPI** вместе со стандартными объявлениями типов Python (короткими и интуитивно понятными), вы получаете:
+
+* Поддержку редактора (проверку ошибок, автозаполнение и т.п.)
+* "<abbr title="преобразование строк из HTTP-запроса в типы данных Python">Парсинг</abbr>" данных
+* Валидация данных
```
Comment (Xewus) [inline docs/ru/docs/tutorial/path-params.md:246]: Вы получаете:
```suggestion
* Валидацию данных
```

File: docs/ru/docs/tutorial/path-params.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+* Аннотацию API и автоматическую документацию
```
Comment (Xewus) [inline docs/ru/docs/tutorial/path-params.md:247]:
```suggestion
* Автоматическую документацию API с указанием типов параметров.
```

File: docs/ru/docs/tutorial/path-params.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+
+И объявлять типы нужно только один раз.
```
Comment (Xewus) [inline docs/ru/docs/tutorial/path-params.md:249]:
```suggestion
И объявлять типы достаточно один раз.
```

File: docs/ru/docs/tutorial/path-params.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+Это, вероятно, является главным заметным преимуществом **FastAPI** по сравнению с альтернативными фреймворками (кроме <abbr title="без учета оптимизаций">сырой</abbr> производительности).
```
Comment (Xewus) [inline docs/ru/docs/tutorial/path-params.md:251]:
```suggestion
Это, вероятно, является главным заметным преимуществом **FastAPI** по сравнению с альтернативными фреймворками (кроме <abbr title="не считая оптимизаций">сырой</abbr> производительности).
```

**Review by Xewus — APPROVED** [review]

**Review by Alexandrhub — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thank you @AGolicyn! 👏 

And thanks for the reviews @Xewus and @Alexandrhub 🙇

**Pull request merged**